packages/perseus/src/components/image-loader.tsx

```diff
@@ -142,11 +142,18 @@ class ImageLoader extends React.Component<Props, State> {
         }
         const staticUrl = getDependencies().staticUrl;
 
+        // If the image is interactive, it should have a role of "button"
+        // to indicate that it is interactive.
+        const imageRole = imgProps.onClick !== null ? "button" : "img";
+
         return (
-            // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions -- TODO(LEMS-2871): Address a11y error
+            // (LEMS-2871) NOTE: This image IS interactive if it requires zooming, and LevelAccess specifically requested
+            // that the role and tabIndex be set here. When users interact with the image on mobile, it will zoom in.
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
We have a ticket to fix this A11Y exceptions, but this is one case that I think we're going to want to keep. It did make me wonder about whether this linter rule could be updated as we're having to write a lot of exceptions for this interactive element.